src/BlazorUI/Bit.BlazorUI.Extras/Styles/extra-components.scss

```
@@ -1,7 +1,8 @@
 ﻿@import "../Components/AccordionList/BitAccordionList.scss";
 @import "../Components/AppShell/BitAppShell.scss";
 @import "../Components/DataGrid/BitDataGrid.scss";
-@import "../Components/DataGrid/Pagination/BitDataGridPaginator.scss";
+@import "../Components/QuickGrid/BitQuickGrid.scss";
```

Stylelint (17.13.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)